content/en/docs/reference/glossary/qos-class.md

```diff
@@ -0,0 +1,24 @@
+---
+title: QoSClass
+id: qosClass
+date: 2019-04-15
+full_link: 
+short_description: >
+  QoSClass provides a way for Kubernetes to classify pods within the cluster into several classes and make decisions about scheduling and eviction.
+
+aka: 
+tags:
+- core-object
+- fundamental
+- architecture
+related:
+ - pod
+
+---
+ QoSClass provides a way for Kubernetes to classify Pods within the cluster into several classes and make decisions about scheduling and eviction.
```

Comment:
If someone doesn't know what QoS is short for, should we explain it?

Reply:
Do you have any proposal here @sftim ?

Reply:
How about rephrasing to use “QoS” on its own, so that this can turn into “QoS (quality of service)”?

Reply:
Alternatively:
```markdown
QoSClass (quality of service class) provides…
```